Assets/Scripts/Managers/RenderSettingsManager.cs

```diff
@@ -79,10 +81,13 @@ public static float daylightIntensity
 	void Awake()
 	{
 		_curSkyboxTintPropertyID = Shader.PropertyToID( "_Tint" );
+		RenderSettings.skybox.SetFloat( "_Rotation", 0f );
 	}
 
 	void Update()
 	{
+		RenderSettings.skybox.SetFloat( "_Rotation", Time.realtimeSinceStartup * _skyboxRotSpeed );
```

Comment:
is this intended to be associated with the time of day at all or does it just spin on its own?